### PR TITLE
CSS layers support for web applications

### PR DESCRIPTION
**Pull request by dmendez**

### CODE REVIEW

File: java/src/main/java/com/genexus/internet/HttpContext.java [hunk 1]
```diff
@@ -164,6 +164,10 @@ public void ajax_sending_grid_row(com.genexus.webpanels.GXWebRow row)
 
 	private static HashMap<String, Messages> cachedMessages = new HashMap<String, Messages>();
 	private String currentLanguage = null;
+	private Vector<Object> userStyleSheetFiles = new Vector<Object>();
+	private String ThemekbPrefix;
```
Comment (ggallotti) [inline java/src/main/java/com/genexus/internet/HttpContext.java:168]: This varname should start with lowecase.

**Review by ggallotti — CHANGES_REQUESTED** [review]
LGTM!
Some minor Issue around VarNames. 
BTW: PR Title should be more descriptive and not include the Issue Number.

**Comment by dmendez** [comment]
> LGTM! Some minor Issue around VarNames. BTW: PR Title should be more descriptive and not include the Issue Number.

**Review by ggallotti — APPROVED** [review]
Thanks! 
Just in case, it seems that latests commits were not included in BETA because of PR conflict.

**Pull request merged**